cmd/koord-manager/main.go

```diff
@@ -224,6 +207,20 @@ func main() {
 	}
 }
 
+func setupControllersWithManager(m manager.Manager) error {
+	for controllerName, addFn := range controllerAddFuncs {
+		if err := addFn(m); err != nil {
+			if kindMatchErr, ok := err.(*meta.NoKindMatchError); ok {
+				klog.Infof("CRD %v is not installed, its controller %s will perform noops!", kindMatchErr.GroupKind, controllerName)
```

Comment:
what is the case where we can ignore to start a controller that some CRD not installed

Reply:
Some controllers expect to decide whether to enable or disable it based on whether there is a CRD in the cluster. But it can be removed first.

Reply:
OK